Frontend/src/clients/RecipeClient.ts

```diff
@@ -3,7 +3,7 @@ export class RecipeClient {
 
      public async getRecipeByName(name: string) {
          try {
-         const response = await fetch(`https://localhost:44328/Recipe/byname/${name}`);
+         const response = await fetch(`http://localhost:5184/Recipe/byname/${name}`);
```

Comment:
bitte die url mit https und port 44328 hier lassen

PS: damit das backend auf der https url läuft, musst du in rider entweder die running configuration "API: https" oder besser "API: IIS Express" benutzen, und nicht "API: http"

![image](https://github.com/SE-TINF22B6/CookHub/assets/77683850/bbc1dcac-19d1-4ddb-8fbd-06fbd08deac7)